### PR TITLE
DM-35364: fix confusing message when authentication fails

### PR DESCRIPTION
**Pull request by wguanicedew**
## Checklist

- [x] ran Jenkins
- [x] added a release note for user-visible changes to `doc/changes`

### CODE REVIEW

File: python/lsst/ctrl/bps/panda/panda_service.py [hunk 1]
```diff
@@ -335,6 +335,10 @@ def get_idds_result(self, ret):
                 status = True
                 result = ret[1][1]
                 error = None
+                if "Authentication no permission" in result:
+                    status = False
```
Comment (timj) [inline python/lsst/ctrl/bps/panda/panda_service.py:339]: Can you put these three lines in the same order as in the other blocks? status/result/error? I know it seems weird but comparing them was made harder by the order being different.

**Review by timj — APPROVED** [review]
This looks consistent with the other code. The way that errors are returned in nested python structures always makes my head hurt a little. I still think that being able to pass `ret` to some python results class that you could then use to hide all this `ret[1][1]`, `ret[1][0]` stuff would help code readability a lot.

**Review by MichelleGower — APPROVED** [review]
Looks fine.  Merge approved.

File: python/lsst/ctrl/bps/panda/panda_service.py (hunk 1 above)
Comment (wguanicedew) [reply python/lsst/ctrl/bps/panda/panda_service.py:339]: updated.

**Comment by wguanicedew** [comment]
For the 'ret' part, we will think how to improve it. I will discuss with PanDA to standardize this part.

**Pull request merged**